src/Engines/QueryBuilderEngine.php

```diff
@@ -378,6 +378,34 @@ protected function compileQuerySearch($query, $column, $keyword, $relation = 'or
         $column = $this->castColumn($column);
         $sql    = $column . ' LIKE ?';
 
+        /**
+         * Patch for fix about ambiguous field
+         *
+         * Ambiguous field error will appear
+         * when query use join table and search with keyword.
+         */
+        // Remove backtick that appear from MYSQL query.
+        $column = str_replace('`', '', $column);
```

Comment:
Not sure but maybe we could try replacing those 4 symbols with empty string? Will check this further on my oracle project when I had the chance. Thanks!

```php
$column = str_replace(['`[]"'], '', $column);
```